src/BasePrivacyBlock.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace luya\privacy;
+
+use luya\cms\base\PhpBlock;
+
+/**
+ * Base Privacy Block
+ *
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.0
+ */
+abstract class BasePrivacyBlock extends PhpBlock
+{
+    /**
+     * @inheritdoc
+     */
+    public function getViewPath()
+    {
+        return '@privacy/views/blocks';
```

Comment:
We should not work with the fix module id, instead use this:

https://github.com/luyadev/luya-bootstrap3/blob/master/src/BaseBootstrap3Block.php#L19